chef/cookbooks/ceph/recipes/glance.rb

```diff
@@ -19,6 +19,7 @@
   mode 0640
 end
 
-execute "create new pool #{glance_pool}" do
-  command "ceph osd pool create #{glance_pool} 64"
+ceph_pool glance_pool do
+  pool_name glance_pool
+  pg_num 128
```

Comment:
Ups, I did it by accidentm, this value will be calculated automatically in this https://github.com/crowbar/barclamp-ceph/pull/92